app/graphql/connections/hosts_connection.rb

```diff
@@ -0,0 +1,6 @@
+Connections::HostsConnection = Types::Host.define_connection do
```

Comment:
I was checking above, but unfortunately I wasn't able to provide workable solution. I've got problems with defining it and get some errors while testing. I came with another solution: 

We can create `Types::BaseObject` class which inherit from `GraphQL::Schema::Object`:

```
module Types
  class BaseObject < GraphQL::Schema::Object
    def self.define_connection(**args, &block)
      super(**args) do
        block&.call

        field :totalCount do
          type types.Int
          resolve ->(obj, _, _) { obj.nodes.size }
        end
      end
    end
  end
end
```

Then: 

- use it for defining all types: `class ModelType < Types::BaseObject`
- get rid of `/connections`
- use connections this way: `field :models, Types::ModelType.connection_type, function: Queries::PluralField.new(type: Types::ModelType, model_class: Model)`

With that we do not have to define custom connections and we follow default usage of connection fields: http://graphql-ruby.org/relay/connections.html#connection-fields

I tested it locally and works fine.
WDYT?